### PR TITLE
CHE-95. Fix wrong scrolling in editor at using code assistant

### PR DESCRIPTION
**Pull request by RomanNikitenko**
Signed-off-by: Roman Nikitenko rnikitenko@codenvy.com

### CODE REVIEW

**Comment by RomanNikitenko** [comment]
@azatsarynnyy WDYT?

**Comment by azatsarynnyy** [comment]
Looks OK

File: plugins/plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionDocument.java [hunk 1]
```diff
@@ -36,6 +35,9 @@
  */
 public class OrionDocument extends AbstractEmbeddedDocument {
 
+    /** Default count lines before specified line number at the top of the editor's view */
```
Comment (azatsarynnyy) [inline plugins/plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionDocument.java]: As for me the next comment would be more clear:

> The maximum number of lines that may be visible at the top of the text view after setting selection range.

But I do not insist )

**Comment by azatsarynnyy** [comment]
Looks OK

**Comment by vparfonov** [comment]
ok

**Pull request merged**